src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1132,6 +1132,44 @@ private AuthenticationResult acquireTokenLocalCall(final CallbackHandler callbac
     private boolean promptUser(PromptBehavior prompt) {
         return prompt == PromptBehavior.Always || prompt == PromptBehavior.REFRESH_SESSION;
     }
+    
+    /**
+     * App needs to give permission to AccountManager to use broker.
+     */
+    private boolean verifyManifestPermissions() {
+        PackageManager pm = mContext.getPackageManager();
```

Comment:
pm could be defined as final. 
also nitpick: I would like to see pm named as packageManager, it's more readable.